src/code128.ps.src

```diff
@@ -396,6 +398,171 @@ begin
 
     encoding (new) eq {
 
+        % Include pseudo characters for GS1-128 Composite linkage identifiers
+        /seta <<fn3 96 fn2 97 fn1 102 stp 106 lka 100 lkc  99>> def
+        /setb <<fn3 96 fn2 97 fn1 102 stp 106 lka  99 lkc 101>> def
+        /setc <<              fn1 102 stp 106 lka 101 lkc 100>> def
+
+        /text msglen string def
+        0 1 msglen 1 sub {
+            /i exch def
+            text i msg i get dup 0 lt { pop 32 } if put
+        } for
+
+        65 dict begin
```

Comment:
A new dictionary doesn't seem necessary. Is there some reason for it?

Reply:
I'll add a feature toggle to Maxicode as well, shall I?

Reply:
Please drop the dictionary for now. It only really provides "side-effect" protection, which isn't a big deal.

We can reconsider the coding style as part of a more general refresh if/when such a thing happens.